apps/admin/components/contract/PreSeason/PreSeasonNFTView.tsx

```diff
@@ -15,11 +16,12 @@ function ContractLink({
   title: string;
   subtitle?: string;
 }) {
+  const chainName = scoutProtocolChain.name.toLowerCase();
   return (
     <Box gap={1} display='flex' flexDirection='column'>
       <Typography variant='h6'>{title}</Typography>
       <Box sx={{ minHeight: '40px' }}>{subtitle && <Typography variant='body2'>{subtitle}</Typography>}</Box>
-      <Link href={`https://optimism.blockscout.com/${linkType}/${address}`} target='_blank'>
+      <Link href={`https://${chainName}.blockscout.com/${linkType}/${address}`} target='_blank'>
```

Comment:
since we have multiple chains per season i am pretty sure this needs to be passed in. we are passing in an address to this component, so it shouldnt be looking up other details like network from a global

Reply:
good catch. updated